lib/Controller/AssistantApiController.php

```diff
@@ -254,6 +254,29 @@ public function shareOutputFile(int $ocpTaskId, int $fileId): DataResponse {
 		}
 	}
 
+	/**
+	 * Save an output file
+	 *
+	 * Save a file that was produced by a task
+	 *
+	 * @param int $ocpTaskId The task ID
+	 * @param int $fileId The file ID
+	 * @return DataResponse<Http::STATUS_OK, array{shareToken: string}, array{}>|DataResponse<Http::STATUS_NOT_FOUND, array{error: string}, array{}>
+	 *
+	 * 200: The file was saved
+	 * 404: The file was not found
+	 */
+	#[NoAdminRequired]
+	public function saveOutputFile(int $ocpTaskId, int $fileId): DataResponse {
+		try {
+			$info = $this->assistantService->saveOutputFile($this->userId, $ocpTaskId, $fileId);
+			return new DataResponse($info);
+		} catch (\Exception $e) {
+			$this->logger->debug('Failed to save assistant output file', ['exception' => $e]);
```

Comment:
Would it be more useful to log this as an error and not debug?

Reply:
Nice catch. Done.